### PR TITLE
Added Night Owl Theme

### PR DESCRIPTION
**Pull request by CrHasher**
This pull adds a new theme named Night Owl based on the Visual Studio Code theme by Sarah Drasner.

**Note**: The image for the theme is missing from the commit because I would need the address to the Theme Testing server, or someone who can make an image based on this Night Owl theme.

### CODE REVIEW

File: source/help/settings/theme-colors.rst [hunk 1]
```diff
@@ -160,3 +160,13 @@ Discord Dark Theme
...
+
+Night Owl Dark theme
+^^^^^^^^^^^^^^^^^^^^
+
+.. image:: ../../images/nightOwlDark.PNG
+  :alt: nightOwlDark
+
+.. code-block:: none
+
+  {"sidebarBg":"#011627","sidebarText":"#d6deeb","sidebarUnreadText":"#d6deeb","sidebarTextHoverBg":"#1d3b53","sidebarTextActiveBorder":"#ff2c83","sidebarTextActiveColor":"#82aaff","sidebarHeaderBg":"#1d3b53","sidebarHeaderTextColor":"#d6deeb","onlineIndicator":"#addb67","awayIndicator":"#ffbc42","dndIndicator":"#f74343","mentionBg":"#7e57c2","mentionBj":"#011627","mentionColor":"#145dbf","centerChannelBg":"#011627","centerChannelColor":"#d6deeb","newMessageSeparator":"#ff8800","linkColor":"#2389d7","buttonBg":"#166de0","buttonColor":"#011627","errorTextColor":"#fd5960","mentionHighlightBg":"#ffe577","mentionHighlightLink":"#166de0","codeTheme":"solarized-dark"}
```
Comment (esethna) [inline source/help/settings/theme-colors.rst:172]: @CrHasher please update mentionBj > mentionBg

cc// @justinegeffen

**Review by esethna — CHANGES_REQUESTED** [review]
This is an awesome theme!

File: source/help/settings/theme-colors.rst (hunk 1 above)
Comment (esethna) [reply source/help/settings/theme-colors.rst:172]: @CrHasher what server version were these values generated on? I'm seeing bugs in the theme when I try this myself

image

Can you try on https://community.mattermost.com/ so we are sure the theme works on our newest versions?

**Review by esethna — CHANGES_REQUESTED** [review]

**Comment by CrHasher** [comment]
Just updated the mentions background the rest of the theme is based on the other themes in the themes color page ex. OneDark, Discord and so on. All those themes have duplicate mentionBg fields in the json. Hope that is ok, the theme was tested and works on https://community.mattermost.com/

**Comment by CrHasher** [comment]
Also note that the #FF2C83 color you see in some places indicates unread and active channels. See this link for details on colors used: https://github.com/VonHeikemen/night-owl-sublime-scheme

File: source/help/settings/theme-colors.rst [hunk 2]
```diff
@@ -160,3 +160,13 @@ Discord Dark Theme
 .. code-block:: none
 
   {"sidebarBg":"#2f3136","sidebarText":"#ffffff","sidebarUnreadText":"#ffffff","sidebarTextHoverBg":"#33363c","sidebarTextActiveBorder":"#66cfa0","sidebarTextActiveColor":"#ffffff","sidebarHeaderBg":"#27292c","sidebarHeaderTextColor":"#ffffff","onlineIndicator":"#43b581","awayIndicator":"#faa61a","dndIndicator":"#f04747","mentionBg":"#6e84d2","mentionBg":"#6e84d2","mentionColor":"#ffffff","centerChannelBg":"#36393f","centerChannelColor":"#dddddd","newMessageSeparator":"#6e84d2","linkColor":"#2095e8","buttonBg":"#43b581","buttonColor":"#ffffff","errorTextColor":"#ff6461","mentionHighlightBg":"#3d414f","mentionHighlightLink":"#6e84d2","codeTheme":"monokai"}
+
+Night Owl Dark theme
+^^^^^^^^^^^^^^^^^^^^
+
+.. image:: ../../images/nightOwlDark.PNG
```
[the remark this answers is not in the document]
Comment (esethna) [reply source/help/settings/theme-colors.rst:167]: You can add it to the source/images folder on your branch :)

**Review by esethna — APPROVED** [review]

**Review by cwarnermm — APPROVED** [review]
Looks good to me!

**Pull request merged**